cdap-common/bin/service

```diff
@@ -72,6 +72,10 @@ done
 # Load user-modifiable configuration
 test -f "${CDAP_CONF}"/cdap-env.sh && source "${CDAP_CONF}"/cdap-env.sh
 
+# Set JAVA_HEAPMAX from variable defined in JAVA_HEAP_VAR
+JAVA_HEAPMAX=${!JAVA_HEAP_VAR}
```

Comment:
which means we should probably just remove it from common-env.sh as well (default 128m is useless anyway)